gcc/testsuite/gdc.test/runnable/testsafe.d

```diff
@@ -215,6 +215,8 @@ void inlineasm()
             static assert(!__traits(compiles, { asm { "nop"; } }() ));
         else version(ARM)
             static assert(!__traits(compiles, { asm { "nop"; } }() ));
+        else version(AArch64)
+            static assert(!__traits(compiles, { asm { "nop"; } }() ));
```

Comment:
Likewise here, `asm { ""; }` should be rejected as well, and is not platform dependent.

Reply:
done.